src/main/kotlin/de/bund/digitalservice/useid/config/SecurityConfig.kt

```diff
@@ -25,9 +26,10 @@ class SecurityConfig(
     @Bean
     fun securityFilterChain(http: HttpSecurity): SecurityFilterChain {
         return http.authorizeHttpRequests()
-            .requestMatchers(HttpMethod.GET, "$IDENTIFICATIONS_BASE_PATH/*/transaction-info").permitAll()
```

Comment:
We can remove this, because we are permitting all requests anyway, correct?

Reply:
Yes, at least that's my understanding of it.